tests/Integration/Doctrine/Repository/TaxonRepositoryTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file was created by developers working at BitBag
+ * Do you need more information about us and what we do? Visit our https://bitbag.io website!
+ * We are hiring developers from all over the world. Join us and start your new, exciting adventure and become part of us: https://bitbag.io/career
+*/
+
+declare(strict_types=1);
+
+namespace Tests\BitBag\SyliusVueStorefront2Plugin\Integration\Doctrine\Repository;
+
+use BitBag\SyliusVueStorefront2Plugin\Doctrine\Repository\TaxonRepository;
+use Faker\Factory;
+use Sylius\Component\Core\Model\TaxonInterface;
+use Sylius\Component\Taxonomy\Model\TaxonTranslation;
+
+final class TaxonRepositoryTest extends AbstractRepositoryTest
+{
+    public function testCreatesChildrenByChannelMenuTaxonQueryBuilder(): void
```

Comment:
I would use the lchrusciel ApiTestCase library to define the better fixtures, and then do better matches than checking count of taxons